<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConservationDialogBase</class>
 <widget class="QDialog" name="ConservationDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Indian WAtershed Management support System</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="Conservation_Module">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="CM">
      <attribute name="title">
       <string>Conservation_Module</string>
      </attribute>
      <widget class="QPushButton" name="PB_CM_5">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>140</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
      <widget class="QLabel" name="LB_CM_8">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>230</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Parameter Files</string>
       </property>
      </widget>
      <widget class="QPushButton" name="PB_CM_8">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>230</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
      <widget class="QPushButton" name="PB_CM_2">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>50</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
      <widget class="QPushButton" name="PB_CM_4">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>110</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
      <widget class="QLabel" name="LB_CM_1">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Rainfall</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
      <widget class="QLabel" name="LB_CM_9">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>290</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Output Folder</string>
       </property>
      </widget>
      <widget class="QLabel" name="LB_CM_5">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>140</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Land Use</string>
       </property>
      </widget>
      <widget class="QLabel" name="LB_CM_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>80</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Flow Accumulation</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="PG_CM">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>480</y>
         <width>331</width>
         <height>31</height>
        </rect>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
      <widget class="QPushButton" name="PB_CM_1">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>20</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
      <widget class="QPushButton" name="PB_CM_3">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>80</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
      <widget class="QPushButton" name="MR_CM">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>480</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Model Run</string>
       </property>
      </widget>
      <widget class="QLabel" name="LB_CM_4">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>110</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Soil Texture</string>
       </property>
      </widget>
      <widget class="QLabel" name="LB_CM_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Slope</string>
       </property>
      </widget>
      <widget class="QPushButton" name="PB_CM_9">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>290</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
      <widget class="QLabel" name="LB_CM_6">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>170</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Aquifer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="PB_CM_6">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>170</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="LE_CM_1">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>20</y>
         <width>231</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="LE_CM_2">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>50</y>
         <width>231</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="LE_CM_3">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>80</y>
         <width>231</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="LE_CM_4">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>110</y>
         <width>231</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="LE_CM_5">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>140</y>
         <width>231</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="LE_CM_6">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>170</y>
         <width>231</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="LE_CM_8">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>230</y>
         <width>231</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="LE_CM_9">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>290</y>
         <width>231</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="LB_CM_10">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>350</y>
         <width>231</width>
         <height>21</height>
        </rect>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Choose conservation measure:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
      <widget class="QLabel" name="LB_CM_11">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>370</y>
         <width>201</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>(Recommended: All)</string>
       </property>
      </widget>
      <widget class="QComboBox" name="CB_CM">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>350</y>
         <width>271</width>
         <height>27</height>
        </rect>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="maxCount">
        <number>64</number>
       </property>
       <item>
        <property name="text">
         <string>All</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>contour_bunding</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>graded_bunding</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>contour_trenching</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>check_dam</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>gully_plug</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>bench_terracing</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>percolation_tank</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ditch_and_furrow</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>farm_pond</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>conservation_ditches</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ground_water_dam</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>gravity_head_recharge_well</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>injection_well</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>recharge_shaft</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>recharge_pit</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>fracture_seal_cementation</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="LE_CM_10">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>420</y>
         <width>271</width>
         <height>31</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Default is 0.85.</string>
       </property>
       <property name="placeholderText">
        <string>0.85 to 0.9 recommended.</string>
       </property>
      </widget>
      <widget class="QLabel" name="LB_CM_17">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>420</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Suitability threshold:</string>
       </property>
      </widget>
      <widget class="QLabel" name="LB_CM_7">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>200</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Boundary</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="LE_CM_7">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>200</y>
         <width>231</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="PB_CM_7">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>200</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
